logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/user/GetUserInfoUseCaseTest.kt

```diff
@@ -113,8 +129,80 @@ class GetUserInfoUseCaseTest {
             .wasInvoked(once)
     }
 
+
+    @Test
+    fun givenAUserWithNoTeam_WhenGettingDetails_thenShouldReturnSuccessResultAndDoNotRetrieveTeam() = runTest {
+        // given
+        given(userRepository)
+            .suspendFunction(userRepository::getKnownUser)
+            .whenInvokedWith(eq(userId))
+            .thenReturn(flowOf(OTHER.copy(team = null)))
+
+        given(teamRepository)
+            .suspendFunction(teamRepository::getTeam)
+            .whenInvokedWith(any())
+            .thenReturn(flowOf(team))
```

Comment:
Can we use the Arrangement pattern as we agreed on previous collective meeting? That way we will make sure we don't continuously copy/paste these setup configs, and that we can reuse them and make tests shorter and more flexible.

Reply:
done